provider/hetzner/hetzner.go

```diff
@@ -0,0 +1,209 @@
+/*
+	by Vladimir Smagin, 2020
```

Comment:
Big no, this has to change 🙂 . PTAL at the existing files

Reply:
done for both files